site/src/pages/docs/client-decorator.mdx

```diff
@@ -169,6 +169,56 @@ preview because <type://ContentPreviewingClient> is evaluated first for the HTTP
 For <type://RetryingClient>, please check out 
 [`RetryingClient` with logging](/docs/client-retry#retryingclient-with-logging).
 
+## Handling responses
```

Comment:
How about removing this and just using the below section title?
`## Modifying response`

Reply:
@minwoox Do you mean that I should make `Modifying response` and `Reacting to or logging` into `##` sections?

Reply:
Yes, I thought they are not related so we can put them into `##` sections. 🤔

Reply:
(Done in previous commit)